### PR TITLE
Update documentations.

### LINKED ISSUE #1: Update docs
We need to update docs, provide usage examples and anything more

### PR DESCRIPTION
**Pull request by singulared**
Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,48 @@ This crate works with Cargo and can be found on
 actix-web-validator = "0.1"
 ```
 
+## Supported `actix_web::web` extractors:
+* `web::Json`
+* `web::Query`
+* `web::Path`
```
Comment (AndreyErmilov) [inline README.md:30]: We haven't finished PathValidator yet. Maybe we should remove it from Readme?

File: README.md [hunk 2]
```diff
@@ -24,6 +24,48 @@ This crate works with Cargo and can be found on
...
+
+#[derive(Deserialize, Validate)]
+pub struct AuthRequest {
+    #[validate(range(min = 1000, max = 9999))]
+    id: u64,
+    response_type: ResponseType,
+}
+
+// Use `Query` extractor for query information (and destructure it within the signature).
+// This handler gets called only if the request's query string contains a `username` field.
```
Comment (AndreyErmilov) [inline README.md:57]: Maybe `if the request's query string contains id and response_type fields`?

File: src/lib.rs [hunk 3]
```diff
@@ -1,9 +1,47 @@
...
+//!
+//! #[derive(Deserialize, Validate)]
+//! pub struct AuthRequest {
+//!     #[validate(range(min = 1000, max = 9999))]
+//!     id: u64,
+//!     response_type: ResponseType,
+//! }
+//!
+//! // Use `Query` extractor for query information (and destructure it within the signature).
+//! // This handler gets called only if the request's query string contains a `username` field.
```
Comment (AndreyErmilov) [inline src/lib.rs:29]: Same misspell with `username`

File: src/query.rs [hunk 4]
```diff
@@ -30,7 +66,45 @@ impl Default for QueryConfig {
...
+///
+/// #[derive(Deserialize, Validate)]
+/// pub struct AuthRequest {
+///     #[validate(range(min = 1000, max = 9999))]
+///     id: u64,
+///     response_type: ResponseType,
+/// }
+///
+/// // Use `Query` extractor for query information (and destructure it within the signature).
+/// // This handler gets called only if the request's query string contains a `username` field.
```
Comment (AndreyErmilov) [inline src/query.rs:97]: Same `username`

File: src/json.rs [hunk 5]
```diff
@@ -37,9 +74,48 @@ impl<T> Deref for ValidatedJson<T> {
     }
 }
 
+/// Json extractor. Allow to extract typed information from request's
```
[the remark this answers is not in the document]
Comment (AndreyErmilov) [reply src/json.rs:77]: I mean lines 19 and 80

[the remark this answers is not in the document]
Comment (singulared) [reply src/json.rs:77]: The same. Implementation `trait FromRequest` are main functionality of `ValidatedJson`. Why not cover it with same docstrings?

**Pull request merged**